charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -88,6 +88,7 @@ spec:
               value: "{{ .Values.api.inviteMode }}"
             {{- if eq .Values.api.inviteMode "email" }}
             {{- $check := required ".Values.api.email needs to be configured with .Values.api.inviteMode set to \"email\"" .Values.api.email -}}
+            {{- if not (or .Values.api.sendgrid.secretRef .Values.api.sendgrid.apiKey) }}
```

Comment:
hm, is this condition needed?

Reply:
yes, so it does not run the checks on the smtp values as we will be using sendgrid instead.